src/NUnitFramework/framework/nunit.framework-netstandard13.project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "supports": {},
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
```

Comment:
I get a raft of warnings from this (and the .NET Standard 1.6) project referencing version 1.6.0 here, which go away when I upgrade this to 1.6.1. I _think_ that's the right solution to the problem?

> Warning		Detected package downgrade: System.Globalization from 4.3.0 to 4.0.11 
 nunitlite-runner-netstandard16 (>= 1.0.0) -> nunitlite-netstandard16 (>= 1.0.0) -> NETStandard.Library (>= 1.6.0) -> System.Net.Primitives (>= 4.0.11) -> runtime.unix.System.Net.Primitives (>= 4.3.0) -> System.Globalization (>= 4.3.0) 
 nunitlite-runner-netstandard16 (>= 1.0.0) -> nunitlite-netstandard16 (>= 1.0.0) -> NETStandard.Library (>= 1.6.0) -> System.Globalization (>= 4.0.11)